src/addons/transitions/ReactCSSTransitionGroupChild.js

```diff
@@ -77,6 +79,10 @@ var ReactCSSTransitionGroupChild = React.createClass({
       CSSCore.removeClass(node, className);
       CSSCore.removeClass(node, activeClassName);
 
+      if (childRef.componentDidTransition) {
+        childRef.componentDidTransition(animationType);
```

Comment:
It is if I'm not mistaken. This call to `componentWillTransition` in the [the `endListener` callback](https://github.com/GuptonMarrsInternational/react/blob/9b697705e13052bb93b2307b3de8db4834aa5f63/src/addons/transitions/ReactCSSTransitionGroupChild.js#L72), which is called after the transition ends.